src/components/form/field-controls/checkbox-field/index.tsx

```diff
@@ -13,11 +12,10 @@ const CheckboxField = ({
 	label,
 	labelFontWeight,
 	name,
-	onChange = () => null, // Default value makes this easier to invoke w/o extra logic
-	onClick,
+	onClick = () => void 0,
+	onChange = () => void 0,
```

Comment:
TIL about `void 0` 😄

Reply:
I think the type definition for `onChange` didn't like returning `null`, and eslint didn't like `() => {}`, so `() => void 0` it was 🙃